examples/schema_site_external_epg/main.tf

```diff
@@ -11,27 +11,106 @@ provider "mso" {
   password = "" # <MSO pwd>
   url      = "" # <MSO URL>
   insecure = true
+  platform = "nd"
+}
+
+resource "mso_site" "test_site" {
+  name             = "test_site"
+  username         = "" # <APIC username>
+  password         = "" # <APIC pwd>
+  apic_site_id     = "105"
+  urls             = ""# <APIC site url>
+  location = {
+    lat  = 78.946
+    long = 95.623
+  }
+}
+
+resource "mso_tenant" "tenant1" {
+  name          = "test_tenant"
+  display_name  = "test_tenant"
+  description   = "DemoTenant"
+  site_associations {
+    site_id     = mso_site.test_site.id
+  }
+}
+
+resource "mso_schema" "schema1" {
+  name          = "test_schema"
+  template_name = "Template1"
+  tenant_id     = mso_tenant.tenant1.id
+}
+
+resource "mso_schema_template_vrf" "vrf1" {
+  schema_id       = mso_schema.schema1.id
+  template        = mso_schema.schema1.template_name
+	name            = "vrf1"
+	display_name    = "vrf1"
+}
+
+resource "mso_schema_template_l3out" "template_l3out" {
+  schema_id             = mso_schema.schema1.id
+  template_name         = mso_schema.schema1.template_name
+  l3out_name            = "l3out1"
+  display_name          = "l3out1"
+  vrf_name              = mso_schema_template_vrf.vrf1.id
+  vrf_schema_id         = mso_schema_template_vrf.vrf1.schema_id
+  vrf_template_name     = mso_schema_template_vrf.vrf1.template
 }
 
 resource "mso_schema_template_external_epg" "template_externalepg" {
-	schema_id           = "5f043b3b2c0000f47e812a0b"
-	template_name       = "Template1"
-	external_epg_name   = "temp_epg"
-    external_epg_type   = "cloud"
-	display_name        = "temp_epg"
-	vrf_name            = "Myvrf"
-    anp_name            = "ap1"
-    l3out_name          = "temp"
-    site_id             = ["5c7c95d9510000cf01c1ee3d"]
-    selector_name       = "check02"
-    selector_ip         = "12.23.34.45"
-}
-
-resource "mso_schema_site_external_epg_selector" "sel1" {
-  schema_id = "${mso_schema_template_external_epg.template_externalepg.schema_id}"
-  template_name = "${mso_schema_template_external_epg.template_externalepg.template_name}"
-  site_id = "${mso_schema_template_external_epg.template_externalepg.site_id}"
-  external_epg_name = "${mso_schema_template_external_epg.template_externalepg.external_epg_name}"
-  name = "second"
-  ip = "12.25.70.50"
-}
\ No newline at end of file
+	schema_id             = mso_schema.schema1.id
+	template_name         = mso_schema.schema1.template_name
+	external_epg_name     = "temp_epg"
+	display_name          = "temp_epg"
+	vrf_name              = mso_schema_template_vrf.vrf1.id
+  vrf_schema_id         = mso_schema_template_vrf.vrf1.schema_id
+  vrf_template_name     = mso_schema_template_vrf.vrf1.template
```

Comment:
Can you align those with the other lines or use terraform fmt (https://www.terraform.io/docs/cli/commands/fmt.html) to do it for you.